project/password/templates/registration/password_change_complete.html

```diff
@@ -0,0 +1,8 @@
+{%extends 'base.html' %}
+
+{%block content%}
+<h2>Your password has been changed</h2>
+
+<a href="/">Profile</a>
+
+{%endblock%}
```

Comment:
Lint: add newline at end of file.

Reply:
kindly explain this lint statement add newline at the end of file